Mathlib/MeasureTheory/Function/L1Space/HasFiniteIntegral.lean

```diff
@@ -117,39 +117,73 @@ theorem HasFiniteIntegral.mono {f : α → β} {g : α → γ} (hg : HasFiniteIn
       lintegral_mono_ae (h.mono fun a h => ofReal_le_ofReal h)
     _ < ∞ := hg
 
+-- XXX: this naming scheme is non-ideal!
+theorem HasFiniteIntegral.mono_e {f : α → ε} {g : α → ε'} (hg : HasFiniteIntegral g μ)
```

Comment:
Probably follow the naming scheme of [MeasureTheory.eLpNorm_mono_enorm](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Function/LpSeminorm/Basic.html#MeasureTheory.eLpNorm_mono_enorm) for now.

Reply:
I agree, we should prefer readability to saving keystrokes.

Reply:
Done!